### DISCUSSION TITLE
How to put CSS code in CSS files generated by JCB? (admin and front)

### DISCUSSION THREAD

**Discussion by fred-the-coder** [joomengine/Joomla-Component-Builder#640] [Q&A] [answered]
Hi,

In assests folder (admin and front), a subfolder css is generated with css files.

How to, from JCB, add CSS code in them?

Thank you

**Comment by laurentgarin** [comment, marked as answer]
Hi,

In site views > your site view > JavaScript & CSS tav > Add CSS *

**Reply by fred-the-coder** [reply]
Mmmm, I have no site view defined...indeed, it is an admin view, with "create edit site view" set...

And my question is related to CSS files in admin and front...do they have the same content?

**Reply by ohrionmartin** [reply]
Hi @fred-the-coder 

In admin views > your admin view > CSS tab > Add CSS to either the edit view or the list view, JCB will create two css files with the same content on compilation, one will be found in admin/components/your_component/assets/css/your_view.css and the other one will be found in components/your_component/assets/css/your_view(created from admin view).css

**Comment by fred-the-coder** [comment]
@ohrionmartin thank you for your answer.
OK got it...
Not sure it is efficient as Admin And Frontend are often different in terms of UI...will see
